clang/lib/DPCT/ASTTraversal.cpp

```diff
@@ -197,5 +198,7 @@ REGISTER_RULE(CuDNNAPIRule, PassKind::PK_Migration, RuleGroupKind::RK_DNN)
 
 REGISTER_RULE(NVSHMEMRule, PassKind::PK_Migration, RuleGroupKind::RK_NVSHMEM)
 
+REGISTER_RULE(CUTENSORRule, PassKind::PK_Migration, RuleGroupKind::RK_CUTENSOR)
```

Comment:
Maybe CUTensorRule is better than CUTENSORRule?
```suggestion
REGISTER_RULE(CUTensorRule, PassKind::PK_Migration, RuleGroupKind::RK_CUTENSOR)
```
If you buy in to it, please also update other places.

Reply:
Done